cluster-autoscaler/cloudprovider/aws/aws_cloud_provider.go

```diff
@@ -227,6 +228,11 @@ func (asg *Asg) Nodes() ([]string, error) {
 	return asg.awsManager.GetAsgNodes(asg)
 }
 
+// TemplateNodeInfo returns a node template for this node group.
```

Comment:
Description expanded (on the interface). 
Yes, this is for scaling from 0 to non zero.